site-src/v1alpha2/references/cross-namespace-references.md

```diff
@@ -0,0 +1,127 @@
+explicitly allows references to Services from HTTPRoutes in the `foo` namespace.
+
+```yaml
+kind: HTTPRoute
+metadata:
+  name: foo
+  namespace: foo
+spec:
+  rules:
+  - matches:
+    - path: /bar
+    forwardTo:
+      backend:
+      - name: bar
+        namespace: bar
+---
+kind: ReferencePolicy
+metadata:
+  name: bar
+  namespace: bar
+spec:
+  from:
+  - group: networking.gateway.k8s.io
+    kind: HTTPRoute
+    namespace: foo
+  to:
+  - group: core
+    kind: Service
+```
+
+### API design decisions
+While the API is simplistic in nature, it comes with a few notable decisions:
+
+1. Each ReferencePolicy only supports a single From and To section.
+   Additional trust relationships must be modeled with additional ReferencePolicy
+   resources.
+1. Resource names are intentionally excluded from this policy for simplicity
+   and because they rarely provide any meaningful protection. A user that is
+   able to write to resources of a certain kind within a namespace can always
+   rename resources or change the structure of the resources to match a given
+   policy.
+1. A single Namespace is allowed per "From" struct.
+   Although a selector would be more powerful, it encourages unnecessarily
+   insecure configuration.
+
+Please see the [API Specification](https://gateway-api.sigs.k8s.io/references/spec/)
+for the details of the object's behavior.
+
+### Exceptions
+There are some situations where it MAY be acceptable to ignore ReferencePolicy
+in favor of some other security mechanism. This MAY only be done if other
+mechanisms like NetworkPolicy can effectively limit cross-namespace references
+by the implementation.
+
+An implementation choosing to make this exception MUST clearly document that
+ReferencePolicy is not honored by their implementations and detail which
+alternative safeguards are available. Note that this is unlikely to apply to
+ingress implementations of the API and will not apply to all mesh
+implementations.
+
+For an example of the risks involved in cross-namespace references, refer to
+[CVE-2021-25740](https://github.com/kubernetes/kubernetes/issues/103675).
+Implementations of this API need to be very careful to avoid confused deputy
+attacks. ReferencePolicy provides a safeguard for that. Exceptions MUST only
+be made by implementations that are absolutely certain that other equally
+effective safeguards are in place.
+
+### Conformance Level
+
+ReferencePolicy support is a "CORE" conformance level requirement for
+cross-namespace references that originate from the following objects:
+- HTTPRoute
+- TLSRoute
+- TCPRoute
+- UDPRoute
+
+That is, all implemenations MUST use this flow for any cross namespaces
+in any of the core xRoute types, except as noted in the Exceptions section
+above.
+
+Other "ImplementationSpecific" objects and references MUST also use this flow
+for cross-namespace references, except as noted in the Exceptions section
+above.
```

Comment:
More a note for me, but when I implement the Route-Gateway binding GEP, I'll need to update this to discuss the uniqueness of that relationship.